public/app/features/commandPalette/actions/recentScopesActions.ts

```diff
@@ -1,17 +1,18 @@
 import { config } from '@grafana/runtime';
 import { t } from 'app/core/internationalization';
-import { defaultScopesServices } from 'app/features/scopes/ScopesContextProvider';
+import { useScopesServices } from 'app/features/scopes/ScopesContextProvider';
 
 import { CommandPaletteAction } from '../types';
 import { RECENT_SCOPES_PRIORITY } from '../values';
 
 export function getRecentScopesActions(): CommandPaletteAction[] {
-  if (!config.featureToggles.scopeFilters) {
+  const services = useScopesServices();
```

Comment:
@tskarhed Updated this to use the services from the context instead of creating new ones. Does not make a difference now but if there is some state in the future it wouldn't be shared.